lib/livebook/session.ex

```diff
@@ -901,6 +905,8 @@ defmodule Livebook.Session do
       session = self_from_state(state)
 
       with :ok <- Livebook.Tracker.track_session(session) do
+        send(caller_pid, {:started, self(), session})
```

Comment:
Ideally we could return info from init, but the GenServer API doesn't allow that, so we send a message instead, which is effectively the same.

Reply:
There is actually a neat trick which is to use `proc_lib:start_link` to boot the process, use `proc_lib:init_ack` to return and then `:gen_server.enter_loop`, but overkill for this case. This is good as is.

Reply:
Oh, we already do it in the evaluator, but I didn't know about `:gen_server.enter_loop`, I actually like that :D

Reply:
I did the change, but I think I will keep it as is, mostly because `init` doesn't have a return, but it should according to the behaviour.